ygnmi/gnmi.go

```diff
@@ -119,7 +119,9 @@ func (gs *getSubscriber) Send(req *gpb.SubscribeRequest) error {
 	}
 	log.V(gs.client.requestLogLevel).Info(prototext.Format(getReq))
 	resp, err := gs.client.gnmiC.Get(gs.ctx, getReq)
-	if err != nil {
+	if st, ok := status.FromError(err); ok && st.Code() == codes.NotFound { // Make this behave like Subscribe, where non-existent paths don't return values.
```

Comment:
Per the spec of Subscribe, I think we need to do the same with "UNIMPLEMENTED"
https://github.com/openconfig/reference/blob/master/rpc/gnmi/gnmi-specification.md#334-getresponse-behavior-table

I _think_ we can leave INVALID_ARGUMENT alone, but I can double-check with Wen

Reply:
this behavior matches the subscribe spec https://github.com/openconfig/reference/blob/master/rpc/gnmi/gnmi-specification.md#3524-subscriberesponse-behavior-table